packages/functional-tests/tests/signin/signinCached.spec.ts

```diff
@@ -0,0 +1,319 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect, newPagesForSync } from '../../lib/fixtures/standard';
+const password = 'passwordzxcv';
+let email;
+let email2;
+let syncBrowserPages;
+
+test.describe('signin cached', () => {
+  test.beforeEach(async ({ target }) => {
+    syncBrowserPages = await newPagesForSync(target);
+    const { login } = syncBrowserPages;
+    await login.clearCache();
+    email = login.createEmail('sync{id}');
+    email2 = login.createEmail();
```

Comment:
I noticed that in the tests below you create the test accounts. I think you should move that logic here and add destroy them in the `afterEach`. This way the tests properly cleanup themselves after each run.